eclair-core/src/test/scala/fr/acinq/eclair/message/PostmanSpec.scala

```diff
@@ -96,8 +102,12 @@ class PostmanSpec extends ScalaTestWithActorTestKit(ConfigFactory.load("applicat
     assert(target == recipientKey.publicKey)
     waitingForRoute ! MessageRoute(Seq.empty, target)
 
-    val RelayMessage(messageId, _, nextNodeId, _, _, Some(replyTo)) = switchboard.expectMessageType[RelayMessage]
+    val Peer.Connect(nextNodeId, _, replyTo1, _) = switchboard.expectMessageType[Peer.Connect]
     assert(nextNodeId == recipientKey.publicKey)
+    val peerConnection = TestProbe[Any]()
+    val peer = TestProbe[Any]()
+    replyTo1 ! PeerConnection.ConnectionResult.AlreadyConnected(peerConnection.ref.toClassic, peer.ref.toClassic)
+    val Peer.RelayOnionMessage(messageId, _, Some(replyTo)) = peer.expectMessageType[Peer.RelayOnionMessage]
```

Comment:
This is worth extracting to a helper function instead of duplicating it.

Reply:
Done